brightness-mode-explainer.md

```diff
@@ -0,0 +1,131 @@
+# Design Document: the need to control screen brightness
+
+_As discussed in the [DAS WG 2021-11-17](https://www.w3.org/2021/11/17-dap-minutes.html) meeting about Screen Brightness and the Screen Wake Lock API, the idea of this document is to make it easier for the meeting attendees to work on an explainer than if this had been committed to GitHub directly. Once this is done the goal is to move the contents to a proper explainer on GitHub._
+
+Previously:
+- https://github.com/WICG/proposals/issues/17
+- https://github.com/w3c/screen-wake-lock/issues/129
+- https://github.com/w3c/devicesensors-wg/issues/51
+- https://www.w3.org/2021/11/17-dap-minutes.html
+
+## Introduction
+
+There is currently a need for web applications to increase the brightness of a device's screen.
+
+This would help address the following use cases:
+- a user scans QR/barcode under various lighting conditions.
+- remote medical, where increasing the screen brightness could assist in remote examination
```

Comment:
This is a stretch (at least, none of us are medical doctors here)... I suggest we remove this.

Reply:
This came from WICG/proposals#17 so I'd like to confirm with @willmorgan before dropping this line.

Reply:
I am not a medical doctor or healthtech developer, but I can think of various body parts or mobility restrictions where front torch or maximum screen brightness would be useful. Not super compelling though.

Reply:
Done, I've removed this item from the list.